templates/power-off-vm.json

```diff
@@ -59,7 +92,7 @@
                             }
                         },
                         "method": "get",
-                        "path": "/subscriptions/@{encodeURIComponent(parameters('SubscriptionId'))}/resourcegroups/@{encodeURIComponent(parameters('ResourceGroup'))}/providers/Microsoft.Compute/virtualMachines/@{encodeURIComponent(parameters('ResourceVM'))}",
+                        "path": "/subscriptions/@{encodeURIComponent(parameters('SubscriptionId'))}/resourcegroups/@{encodeURIComponent(parameters('ResourceGroup'))}/providers/Microsoft.Compute/virtualMachines/@{encodeURIComponent(parameters('ResourceVM'))}/powerOff",
```

Comment:
This does not look correct. Why do you need to do GET on powerOFF in "Get_virtual_machine_state"

Reply:
Also now that we are sending email on success as well shouldnt we get vm status for 

"runAfter": {
                        "Power_off_virtual_machine": [
"Skipped",
                            "Failed"
""Suuceeded"
                        ]

?

Reply:
Thanks I've resolved the get error. There's no possible scenario that could result in a skip. Priti and I discussed this earlier in the day.